client/wcshipping-migration-admin-notice.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import '../assets/stylesheets/migration_to_wcshipping_admin_notice.scss';
+import FeatureAnnouncement from 'components/migration/feature-announcement';
+import ShippingLabel from 'wcs-client/apps/shipping-label';
+import { setNonce, setBaseURL } from 'wcs-client/api/request';
+
+const container = document.getElementById('wcst_wcshipping_migration_admin_notice_feature_announcement');
+const args = container.dataset.args && JSON.parse( container.dataset.args ) || {};
+setNonce( args.nonce );
+setBaseURL( args.baseURL );
+const ShippingLabelStore = ShippingLabel(args);
+const store = createStore(ShippingLabelStore.getReducer(), ShippingLabelStore.getInitialState());
+
+const wcstWCShippingMigrationNoticeButton = document.getElementById('wcst-wcshipping-migration-notice__click');
+
+// Clicking "Confirm update" will start the migration. This is the same as popping up the modal and clicking the "Update" button there.
+["click", "keydown"].forEach(eventName =>
+	wcstWCShippingMigrationNoticeButton.addEventListener(eventName, () => {
+		// Pop open feature announcement modal.
+		ReactDOM.render(
+			<Provider store={store}>
+				<FeatureAnnouncement />
+			</Provider>,
+			container
+		);
+
+		// Click the update button in the modal to start the migration.
+		const update_button = document.getElementsByClassName('migration__announcement-update-button');
```

Comment:
Maybe it'd be more precise if we used an ID instead?

Reply:
Thanks! I thought about using `ID` but I couldn't find `ID` in https://developer.wordpress.org/block-editor/reference-guides/components/button/#props. 
But! Now that you brought it up again, I wonder if `ID` is "native" so it should just work? Let me give this a try.

Reply:
ID works, updated in 402998bb